packages/manager/modules/nasha/src/nasha.routing.js

```diff
@@ -8,6 +8,16 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
     resolve: {
       breadcrumb: () => NASHA_TITLE,
     },
+    redirectTo: (transition) =>
```

Comment:
As mentioned by Sonarcloud (see: https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=6788&id=ovh_manager&open=AX_cpCPP2APkB2szqB4R), redirectTo is already defined.  Should it be replaced?

Reply:
My bad, merge issue... Fixed :+1: